website/docs/docs/deploy/ci-jobs.md

```diff
@@ -216,7 +216,9 @@ If you're on a Virtual Private dbt Enterprise plan using security features like
 
 When you start a CI job, the pull request status should show as `pending` while it waits for an update from dbt. Once the CI job finishes, dbt sends the status to Azure DevOps (ADO), and the status will change to either `succeeded` or `failed`. 
 
-If the status doesn't get updated after the job runs, check if there are any git branch policies in place that's blocking ADO from receiving these updates. You can find relevant information here:
+If the status doesn't get updated after the job runs, check if there are any git branch policies in place that's blocking ADO from receiving these updates. An example of a branch policy that would result in this issue is enabling "Reset conditions" under "Status checks". If you toggle on "Reset status whenever there are new changes", this will impact dbt's ability to update ADO about the status of your CI job run.
```

Comment:
```suggestion
If the status doesn't get updated after the job runs, check if there are any git branch policies in place blocking ADO from receiving these updates. 

An example of a branch policy resulting in this issue is enabling **Reset conditions** under **Status checks**. If you toggle on **Reset status whenever there are new changes**, this will impact dbt's ability to update ADO about the status of your CI job run.
```